docs/index.rst

```diff
@@ -88,12 +77,6 @@ Advanced Topics
    tutorials/text_prediction/index
    tutorials/tabular_prediction/tabular-multimodal
    tutorials/cloud_fit_deploy/index
-   tutorials/course/index
    api/autogluon.predictor
-   api/autogluon.core
-   api/autogluon.core.space
```

Comment:
Spaces documentation is still useful: https://auto.gluon.ai/stable/api/autogluon.core.space.html#module-autogluon.core.space

Reply:
Removing as it referenced configspace in docs. We can add back afterwards in follow-up.